README.md

```diff
@@ -64,13 +64,14 @@
 ### Setting Up Locally
 
 - Install [Go](https://go.dev/).
+- Install [Docker](https://docs.docker.com/engine/install/).
```

Comment:
Does docker compose get installed along with this or does it have to be installed as a separate component?

Reply:
It does get installed with docker engine.